code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -80,24 +79,23 @@ Bonus
 		var/mob/living/M = A.affected_mob
 		switch(A.stage)
 			if(1)
-				M << "<span class='warning'>You can't taste a thing.</span>"
```

Comment:
It was a useless message, but having a message, or any message in stage 5 will cause it to repeat over, and over, and over again and I felt I should bump any stage 5 messages to stage 4, and so on so forth. Stage 1 had a useless message so I dropped it.